### ISSUE TITLE
New Frontend Template

### ISSUE THREAD

**Issue by kunal-kataria** [arpit456jain/Cool-Front-End-Templates#818]
## Issue Type:

- [ ] Bug Report
- [x] Feature Request
- [x] Documentaion

## **Possible solution**

- I want to add a new Frontend Template.

## **Screenshots**
![preview](https://user-images.githubusercontent.com/72653868/168458024-b965d8dd-8570-423e-8da3-f30c37382330.png)
![preview 2](https://user-images.githubusercontent.com/72653868/168458018-c54cb075-f205-46e8-970e-8c6d78e2bdfd.png)
![preview 3](https://user-images.githubusercontent.com/72653868/168458019-eb7e2eaf-0c2d-41e3-b575-fb2c92fe6729.png)

**Comment by kunal-kataria** [comment]
Can I add a new Template?

**Comment by DEEPIKA-SUK** [comment]
@kunal-kataria Be careful from next time. Don't open the PR until you get assigned in a issue.

**Comment by kunal-kataria** [comment]
@DEEPIKA-SUK My PR is closed but i haven't submitted it yet because my exams were going on. Is it possible to open my PR to submit my template?

**Comment by DEEPIKA-SUK** [comment]
@kunal-kataria create a new PR with all the requested changes completely done. Make the PR by tomorrow otherwise we will close this issue.
